cmake/Modules/Options.cmake

```diff
@@ -475,3 +475,9 @@ endif()
 
 option(SPACK_BUILD "Tweak some installation directories when building via spack" OFF)
 mark_as_advanced(SPACK_BUILD)
+
+option(TIMEMORY_USE_WINSOCK "Include winsock.h with the windows build" OFF)
```

Comment:
Please surround this with `if(WIN32)` so that it doesn't appear as an option on non-Windows builds